torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -267,6 +267,10 @@ struct AddGenericMetadata : public MetadataBase {
       addMetadata("Fwd thread id", std::to_string(op_event.forward_tid_));
       addMetadata("Sequence number", std::to_string(op_event.sequence_number_));
     }
+    if (op_event.record_function_id_ >= 0) {
```

Comment:
Why does it need to check the id >= 0? I think the id starts from 0. 

This is the code to get id:

RecordFunctionHandle next_unique_record_function_handle() {
  static std::atomic<uint64_t> unique_rf_id {1};
  return RecordFunctionHandle(unique_rf_id++);
}

Reply:
oh yea I thought i fixed this actually, to not check > 0, previously the data type was int64_t but now its uint64_t so that fixes this